packages/web-app/app/api/rising/route.ts

```diff
@@ -0,0 +1,14 @@
+import { computeRisingTideCap } from "@/app/_server/risingTide/risingtide";
+import { formatEther } from "viem/utils";
+
+export const revalidate = 0;
+
+BigInt.prototype.toJSON = function() {
+  return this.toString();
+};
+
+export async function GET(_request: Request) {
+  const risingTideCap = await computeRisingTideCap();
+
+  return Response.json({ cap: risingTideCap, capInEther: formatEther(risingTideCap) });
+}
```

Comment:
You're creating a route just for testing? Because in the app we don't need the route, we can just use the server action

Reply:
Yes, I want to be able to call this whenever we make a contribution just to check if this is updating correctly. I'll remove it once we are done